zapcore/json_encoder_impl_test.go

```diff
@@ -223,7 +223,9 @@ func TestJSONEncoderObjectFields(t *testing.T) {
 	}
 
 	for _, tt := range tests {
-		assertOutput(t, tt.desc, tt.expected, tt.f)
+		t.Run(tt.desc, func(t *testing.T) {
+			assertOutput(t, tt.desc, tt.expected, tt.f)
```

Comment:
do we need to pass `tt.desc` through and log them again on failures? it seems like just `t.Run(tt.desc, ...)` is enough.

Reply:
We don't need to pass `tt.desc`, I removed it.